dio/lib/src/dio_event_processor.dart

```diff
@@ -0,0 +1,137 @@
+import 'dart:async';
+
+import 'package:dio/dio.dart';
+import 'package:sentry/sentry.dart';
+// ignore: implementation_imports
```

Comment:
@ueman is that because `sentry_exception_factory` isn't exported?

Reply:
I don't think so. The error message says that the getter do not exist, which is true for 6.3.0 but at the current tip of main it exists. Also, locally I don't get any errors/warnings. Only if I run it against the current stable I get the same warnings.

Reply:
Adding `export 'src/sentry_exception_factory.dart';` to `sentry.dart` fixes the issue, since its a property on SentryOptions, its fine to be there too (Tested it locally).

Reply:
I would say we keep it as is.